frontend/src/routes/Home/Overview/OverviewPage.tsx

```diff
@@ -631,80 +629,78 @@ export default function OverviewPage() {
-                    title={t('Pods')}
-                    description={t('Overview of pod count and status')}
-                    loading={searchLoading}
-                    data={podData}
-                    colorScale={colorThemes.criticalLowSuccess}
-                  />
-                ) : undefined}
-                <AcmDonutChart
-                  title={t('Cluster status')}
-                  description={t('Overview of cluster status')}
-                  loading={!clusterData}
-                  data={clusterData}
-                  colorScale={colorThemes.criticalSuccess}
-                />
-                <AcmDonutChart
-                  title={t('Cluster issues')}
-                  description={t('Overview of cluster issues')}
-                  loading={!policyReportData}
-                  data={policyReportData}
-                  donutLabel={{
-                    title: `${clustersWithIssuesCount}`,
-                    subTitle: clustersWithIssuesCount === 1 ? t('Cluster with issues') : t('Clusters with issues'),
-                  }}
-                  colorScale={colorThemes.criticalImportantModerateLow}
-                />
+          <Stack hasGutter>
+            <AcmMasonry minSize={400} maxColumns={searchError ? 3 : 4}>
+              <AcmDonutChart
+                title={t('Cluster violations')}
+                description={t('Overview of policy violation status')}
+                loading={!complianceData}
+                data={complianceData}
+                colorScale={colorThemes.criticalSuccess}
+              />
+              {!searchError ? (
                 <AcmDonutChart
-                  title={t('Cluster add-ons')}
-                  description={t('Overview of cluster add-ons')}
-                  loading={!clusterAddonData}
-                  data={clusterAddonData}
-                  colorScale={colorThemes.criticalLowUnknownSuccess}
+                  title={t('Pods')}
+                  description={t('Overview of pod count and status')}
+                  loading={searchLoading}
+                  data={podData}
+                  colorScale={colorThemes.criticalLowSuccess}
                 />
-              </AcmMasonry>
-            </Stack>
+              ) : undefined}
+              <AcmDonutChart
+                title={t('Cluster status')}
+                description={t('Overview of cluster status')}
+                loading={!clusterData}
+                data={clusterData}
+                colorScale={colorThemes.criticalSuccess}
+              />
+              <AcmDonutChart
+                title={t('Cluster issues')}
+                description={t('Overview of cluster issues')}
+                loading={!policyReportData}
+                data={policyReportData}
+                donutLabel={{
+                  title: `${clustersWithIssuesCount}`,
+                  subTitle: clustersWithIssuesCount === 1 ? t('Cluster with issues') : t('Clusters with issues'),
```

Comment:
You could pass a `count` interpolation variable and use plural support in the translation library.